strings/traumas.json

```diff
@@ -184,7 +185,8 @@
 	"YOU. STOP.",
 	"REST, MORTALS, TOMORROW IS A LONG DAY.",
 	"YOU MORTALS MAKE ME SICK.",
-	"HONK..."
+	"HONK...",
+	"THERE IS NO SUCH THING AS A COINCIDENCE."
```

Comment:
These lines are used by Godwokem Syndrome and contain keywords (such as stop, sick, rest, honk) that cause the voice of god effect. This line doesn’t contain any voice of god keywords and honestly doesn’t fit with the rest of the lines

Consider checking the voice of god to check out what the keywords are and use one of them in this

Reply:
Whoops, got it, figured they were easter egg lines like the other stuff in the file.